### PR TITLE
Adding a resource

### PR DESCRIPTION
**Pull request by CBID2**
### Purpose
This PR adds a link from a section in OpenSauced's Maintainer course to the [Building Community chapter](https://opensource.guide/building-community/). This resource would help new open source maintainers learn more ways to building and facilitating communities for their open source project.  

- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

### CODE REVIEW

File: _articles/zh-hant/how-to-contribute.md [hunk 1]
```diff
@@ -204,6 +204,7 @@ redirect_from: /zh-tw/how-to-contribute/
 
 * [GitHub 探索](https://github.com/explore/)
 * [First Timers Only](http://www.firsttimersonly.com/)
+* [你的第一個 PR](https://yourfirstpr.github.io/)
```
Comment (jmeridth) [inline _articles/zh-hant/how-to-contribute.md:207]: This link 404s

File: _articles/building-community.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 lang: en
-title: Building Welcoming Communities
```
Comment (jmeridth) [inline _articles/building-community.md:3]: Did you mean to remove the title?

File: _articles/building-community.md [hunk 3]
```diff
@@ -273,4 +273,4 @@ Your tiebreaker should be a last resort. Divisive issues are an opportunity for
 
 ## Community is the ❤️ of open source
 
-Healthy, thriving communities fuel the thousands of hours poured into open source every week. Many contributors point to other people as the reason for working - or not working - on open source. By learning how to tap into that power constructively, you'll help someone out there have an unforgettable open source experience.
+Healthy, thriving communities fuel the thousands of hours poured into open source every week. Many contributors point to other people as the reason for working - or not working - on open source. By learning how to tap into that power constructively, you'll help someone out there have an unforgettable open source experience. If you're looking for more ways to build your open source community, check out the chapter ["Building and Nurturing a Welcoming and Supportive Community" in OpenSauced's "Becoming a Maintainer" course](https://intro.opensauced.pizza/#/becoming-a-maintainer/building-community?id=building-and-nurturing-a-welcoming-and-supportive-community).
```
Comment (jmeridth) [inline _articles/building-community.md:276]: I'm not sure we want to link to a whole other guide.  This is being discussed.

**Review by jmeridth — CHANGES_REQUESTED** [review]
Title was removed and the new link 404s.  Added comments.

File: _articles/building-community.md (hunk 2 above)
Comment (codeo44) [reply _articles/building-community.md:3]: Me?

Comment (CBID2) [reply _articles/building-community.md:3]: No @jmeridth

**Comment by 8012145393085** [comment]
Hello world

File: _articles/zh-hant/how-to-contribute.md (hunk 1 above)
Comment (Sniper1h) [reply _articles/zh-hant/how-to-contribute.md:207]: 😊

File: _articles/building-community.md (hunk 3 above)
Comment (CBID2) [reply _articles/building-community.md:276]: If I may @jmeridth, I have not seen most materials on becoming an open source maintainer presented in a structured way before, and I think alot of people can benefit from it.

Comment (jmeridth) [reply _articles/building-community.md:276]: Our guide does that. I'll get an answer on this tomorrow and let you know. Thank you for your patience.

File: _articles/building-community.md (hunk 2 above)
Comment (jmeridth) [reply _articles/building-community.md:3]: @CBID2 if you didn't mean to remove the title, could we please put it back?

File: _articles/zh-hant/how-to-contribute.md (hunk 1 above)
Comment (jmeridth) [reply _articles/zh-hant/how-to-contribute.md:207]: @CBID2 did you mean to add this link that 404s?  Could you please remove it?

**Review by jmeridth — COMMENTED** [review]
After looking at the commits on this PR and your change, I think your PR consists of commits you didn't make and they have caused unforeseen changes (missing title, 404 link).

However, after discussion with other maintainers, we've decided against adding the opensauced link as it is to your employer, a private company which is currently mentioned in our [CONTRIBUTING guide](https://github.com/github/opensource.guide/blob/main/CONTRIBUTING.md#:~:text=Please%20don%27t%20add%20references%20or%20links%20to%20things%20you%20or%20your%20employer%20have%20created%2C%20as%20others%20will%20do%20so%20if%20they%20appreciate%20it.)

Thank you for your PR and we welcome other changes that do not go against our CONTRIBUTING guide currently.

**Comment by jmeridth** [comment]
Closing. Please refer to previous comment about our [CONTRIBUTING.md guide and links to employer content](https://github.com/github/opensource.guide/blob/main/CONTRIBUTING.md#:~:text=Please%20don%27t%20add%20references%20or%20links%20to%20things%20you%20or%20your%20employer%20have%20created%2C%20as%20others%20will%20do%20so%20if%20they%20appreciate%20it.).

Again, thank you for your PR and we welcome other changes that do not go against our CONTRIBUTING guide currently.

**Pull request closed without merging**